docs/content/architecture/pgadmin4.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "pgAdmin 4"
+date:
+draft: false
+weight: 900
```

Comment:
Ha, how did you come up with this weight? "We may add other stuff, but I want pgAdmin to always be last"?

(Seriously, I was going to suggest going lighter than 900 since the next biggest is, what, 140; but since we can change this weight on any deploy, if we decide we need to move it or make space, we can do that then.)

Reply:
Haha, I did not. That was the weight from the old docs. We can for sure make it lower